OnlineShop/OnlineShop.DAL/Entities/Implementations/Order.cs

```diff
@@ -2,7 +2,15 @@
 
 namespace OnlineShop.DAL.Entities.Implementations;
 
-public class Order:IEntity
+public class Order : IEntity
 {
     public Guid Id { get; set; }
+
+    public required Guid UserId { get; set; }
+    public required DateTime OrderDate { get; set; }
+    public required decimal TotalPrice { get; set; }
```

Comment:
Why do you need this property here? It can be calculated in another way without explicit storage